frontend/smart-kickers-game/src/App.js

```diff
@@ -8,11 +8,14 @@ function App() {
   let [whiteScore, setWhiteScore] = useState(0);
 
   useEffect(() => {
-    const socket = new WebSocket("ws://localhost:3000")
-
+    const socket = new WebSocket.w3cwebsocket('ws://localhost:3006/csc');
```

Comment:
```suggestion
    const socket = new WebSocket('ws://localhost:3006/csc');
```
Why reverting to w3cwebsocket?
This potentially break tests